packages/channels/botpress-channel-web/src/views/web/side/index.jsx

```diff
@@ -101,6 +108,16 @@ export default class Side extends React.Component {
             </g>
           </svg>
         </i>
+        {this.state.showConvoMenu && !this.state.showConvos ? (
+          <div className={style.conversationMenu} style={this.props.fullscreen ? { right: '10px' } : { right: '55px' }}>
+            <div className={style.conversationMenuItem} onClick={::this.handleToggleShowConvos}>
+              Conversations
+            </div>
+            <div className={style.conversationMenuItem} onClick={this.downloadConversation}>
+              Download as *.txt
```

Comment:
let's change it to "Download transcript"

Reply:
Also can it be always visible, i.e. not a menu. Should have its own icon and be a config variable `enableTranscriptDownload` to show/hide the button.

Or we can keep it in a menu, but it doesn't make sense to me that it's in the "showConvo" menu. It's not related at all. I'd rather go with a three-dots menu where we can cram these power features in